script.js

```diff
@@ -1,4 +1,296 @@
+//Initializing Variables
+var mode = "HOME_TURF";
+var player1Array = [];
+var player2Array = [];
+var randomArray = [];
+var myOutputValue = "";
+var randomNumberOnly = 0;
+var randomNumber1 = 0;
+var randomNumber2 = 0;
+var P1FinalNumber = 0;
+var P2FinalNumber = 0;
+var P1_Record = 0;
+var P2_Record = 0;
+var condition = "";
+var LCN_Final_Number_1 = 0;
+var LCN_Final_Number_2 = 0;
+var LCN_Final = 0;
+var P1_max = 0;
+var P2_max = 0;
+var Final_P1_VND = 0;
+var Final_P2_VND = 0;
```

Comment:
Great job thinking through for each player! Now that you've learnt objects, what would you do differently?